metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation/explicit_bucket_histogram.rb

```diff
@@ -19,7 +19,7 @@ class ExplicitBucketHistogram
           # (50.0, 75.0], (75.0, 100.0], (100.0, 250.0], (250.0, 500.0],
           # (500.0, 1000.0], (1000.0, +inf)
           def initialize(
-            aggregation_temporality: :delta,
+            aggregation_temporality: ENV.fetch('OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE', :delta), # the default should be :cumulative, but we haven't implemented it yet
```

Comment:
I'm not sure what the current policy in this repo is; but I think this should be prefixed with `TODO` and have an associated tracking issue

Reply:
Great point! Issue here: https://github.com/open-telemetry/opentelemetry-ruby/issues/1555 

```suggestion
            aggregation_temporality: ENV.fetch('OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE', :delta), # TODO: the default should be :cumulative, see Issue #1555
```